### PR TITLE
add change log for custom spherical shell

### PR DESCRIPTION
**Pull request by LudovicJnnt**
Just adding a change log for the custom spherical shell.

### CODE REVIEW

File: doc/modules/changes/20190530_jeanniot [hunk 1]
```diff
@@ -0,0 +1,4 @@
+New: Extended spherical shell geometry model to 
+include custom mesh schemes. 
+<br>
+(L. Jeanniot and M. Kajan, 2019/05/30)
```
Comment (bangerth) [inline doc/modules/changes/20190530_jeanniot]: We generally spell out first names as well.

**Comment by LudovicJnnt** [comment]
I resolve your comments, thanks!

**Comment by gassmoeller** [comment]
Since this is only documentation I am going to merge.

**Comment by bangerth** [comment]
Related to #3097.

File: doc/modules/changes/20190530_jeanniot [hunk 2]
```diff
@@ -0,0 +1,4 @@
+New: Extended spherical shell geometry model to 
+include custom mesh schemes. 
+<br>
+(Ludovic Jeanniot and Marie Kajan, 2019/05/30)
```
Comment (bangerth) [inline doc/modules/changes/20190530_jeanniot]: This really only lists 2 of the 3 authors of #3097 ;-) ;-)

Comment (LudovicJnnt) [reply doc/modules/changes/20190530_jeanniot]: That is not cool from me to have not listed the most important author of this new feature. 
My apologies for this oversight, I just opened a pull request to rectify the changes log #3143 (related to #3097)  :)

**Pull request merged**